src/content/blam/tags/scenario_structure_bsp/readme.md

```diff
@@ -3,7 +3,14 @@ title: scenario_structure_bsp
 template: tag
 img: bsp.jpg
 imgCaption: The map a30's BSP with all objects removed using `object_destroy_all`
-tagClass: ScenarioStructureBSP
+keywords:
+  - lightmap
+  - radiosity
+  - cluster
+  - plane
+  - weather
+  - polyhedra
+  - bsp
```

Comment:
These are boosted 2x in search results compared to body text. Titles are boosted 3x.